src/main/resources/tags/info/bungee.tag

```diff
@@ -5,4 +5,10 @@ aliases: bungeecord, velocity
 
 The Geyser and Floodgate BungeeCord plugins only need to be installed on the BungeeCord proxy unless you intend to use the Floodgate API. That way, you need Floodgate on the "backend" servers too.
 Installing Floodgate on the backend servers also makes it so that Bedrock players don't have to switch backend servers for their skins to start displaying.
+
+To properly set up Floodgate on the proxy <-> Floodgate on the backend connection, do the following:
+1. Set up player-info-forwarding in the proxy configuration files.
```

Comment:
I'm pretty sure this setting is Velocity only.

Reply:
yea, right, I should adjust it to also include the bungee equivalent.. ip-forward was it iirc